ws/room.go

```diff
@@ -48,11 +47,9 @@ func (r *Room) newSession(host, client xid.ID, rooms *Rooms) {
 		iceHost = []outgoing.ICEServer{{URLs: rooms.addresses("stun", false)}}
 		iceClient = []outgoing.ICEServer{{URLs: rooms.addresses("stun", false)}}
 	case ConnectionTURN:
-		hostPW := util.RandString(20)
-		clientPW := util.RandString(20)
-		hostName := id.String() + "host"
+		hostName, hostPW := rooms.turnServer.Credentials(id.String() + "host")
```

Comment:
Can the Credentials and Allow method call be merged, they are always called in sequence. When calling Credentials it could already register the credentials on the internal turn server or do nothing if the external one is configured.